01_prepare_host.sh

```diff
@@ -109,6 +109,12 @@ if [ ! -f "${IMAGE_NAME}" ] ; then
 fi
 popd
 
+# Pulling images
+for IMAGE_VAR in $(env | grep "_IMAGE=" | grep -o "^[^=]*") ; do
```

Comment:
you need to exclude the _LOCAL_IMAGE here, as they do not exist yet, and the registry is unreachable here

Reply:
Done